extensions/src/main/java/org/apache/falcon/extensions/store/ExtensionStore.java

```diff
@@ -382,14 +382,14 @@ public boolean isExtensionStoreInitialized() {
 
     public List<String> getJobsForAnExtension(final String extensionName) throws FalconException {
         List<ExtensionJobsBean> extensionJobs = metaStore.getJobsForAnExtension(extensionName);
+        List<String> extensionJobNames = new ArrayList<>();
         if (null != extensionJobs && !extensionJobs.isEmpty()){
-            List<String> extensionJobNames = new ArrayList<>();
             for (ExtensionJobsBean extensionJobsBean: extensionJobs) {
                 extensionJobNames.add(extensionJobsBean.getJobName());
             }
             return extensionJobNames;
         } else {
```

Comment:
Unnecessary else. You can just return extensionJobNames outside the if statement.